### ISSUE TITLE
How to Refresh Channels List Screen to filter out specific channels

### ISSUE THREAD

**Issue by muaviyaijaz123** [GetStream/stream-chat-swift#2151]
### Discussed in https://github.com/GetStream/stream-chat-swift/discussions/2135

<div type='discussions-op-text'>

<sup>Originally posted by **muaviyaijaz123** June 30, 2022</sup>
I have my channels list on which I have all of pinned channels like here below:
<img width="363" alt="Screen Shot 2022-06-30 at 5 16 23 PM" src="https://user-images.githubusercontent.com/90596703/176675178-389cc095-a1c1-4c8b-9ca8-3f297f356dba.png">

 --> Query used for pin:
```
         let pinnedChannelListQuery: ChannelListQuery = .init(
            filter: .and([.or([.greater(.lastMessageAt, than: Calendar.current.date(byAdding: .day, value: -1, to: Date())!) , 
   .less(.lastMessageAt, than: Calendar.current.date(byAdding: .day, value: -1, to: Date())!) ]), .containChannelID(
                channelIds: channelID
            )])
           )
```
           
After unpinning , i want to refresh my channels list so the unpinned channel should disappear, but it remains in my list like below although it has been unpinned but it is visually present:
           
<img width="361" alt="Screen Shot 2022-06-30 at 5 17 01 PM" src="https://user-images.githubusercontent.com/90596703/176675712-0decf417-17f9-4523-818c-1f430bcb3a56.png">

So, basically i want to refresh my channels list while remaining on the SAME PAGE after implementing an operation</div>

**Comment by nuno-vieira** [comment]
Hi @muaviyaijaz123!

What do you mean by pinning a channel? StreamChat does not have a pinning channel feature out of the box. Since you don't specify that pinning in the query, it won't change anything. How are you pinning a channel?

Thanks,
Nuno
